js/src/utils/logger.ts

```diff
@@ -32,45 +23,45 @@ export const getLogLevel = (): keyof typeof LOG_LEVELS => {
     : "info";
 };
 
-// Configure winston colors
-winston.addColors(LOG_COLORS);
-
-// Create custom log format
-const logFormat = winston.format.combine(
-  winston.format.timestamp(),
-  winston.format.colorize(),
-  winston.format.printf(({ timestamp, level, message, ...metadata }) => {
-    // Format timestamp for readability
-    const formattedTime = timestamp.slice(5, 22).replace("T", " ");
+const addTimestampToMessage = (message: string): string => {
```

Comment:
The removal of Winston logger for a simpler console-based logger might reduce functionality. Consider:

1. Adding log rotation capability
2. Maintaining log levels consistency
3. Adding log file output option
4. Adding structured logging support

These features are important for production environments. If Winston was removed for bundle size reasons, consider making it an optional dependency.